src/auto/date_time.rs

```diff
@@ -265,9 +270,11 @@ impl DateTime {
         }
     }
 
-    //pub fn to_timeval(&self, tv: /*Ignored*/&mut TimeVal) -> bool {
-    //    unsafe { TODO: call glib_sys:g_date_time_to_timeval() }
-    //}
+    pub fn to_timeval(&self, tv: &mut TimeVal) -> bool {
```

Comment:
This should be `pub fn to_timeval(&self) -> Option<TimeVal>`

Reply:
as manual binding?

Reply:
I'm not sure why gir doesn't autogenerate it as such. @EPashkin ?

But yes, manual binding until gir is fixed

Reply:
A new bug has been found! Yeay!

Reply:
Release just got delayed by another bug. :3